### ISSUE TITLE
Not Work Script 

### ISSUE THREAD

**Issue by ghost** [simsalabim/dsweeper#7]
This script does not work on my computer
The error code is as follows
Can you see what the problem is?
I did run pgrep -f dsweeper command and nothing happened

```
JongYoungs-Mac-Pro:~ jongyoung$ dsweeper -d
Dropbox AppleDouble sweeper is running.
To stop the process run: kill `pgrep -f dsweeper`
child_process.js:401
    throw new TypeError('"cwd" must be a string');
    ^

TypeError: "cwd" must be a string
    at normalizeSpawnArguments (child_process.js:401:11)
    at Object.exports.spawn (child_process.js:488:38)
    at Function.module.exports.daemon (/usr/local/lib/node_modules/dsweeper/node_modules/daemon/index.js:50:31)
    at module.exports (/usr/local/lib/node_modules/dsweeper/node_modules/daemon/index.js:25:20)
    at Object.<anonymous> (/usr/local/lib/node_modules/dsweeper/bin/dsweeper:29:20)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
```

**Comment by simsalabim** [comment]
Hi @pareut and thanks for the bug report. The error you're seeing is [`"throw new TypeError('"cwd" must be a string');"`](https://www.google.com/search?q=throw+new+TypeError(%27%22cwd%22+must+be+a+string%27)%3B&oq=throw+new+TypeError(%27%22cwd%22+must+be+a+string%27)%3B&aqs=chrome..69i57.257j0j7&sourceid=chrome&ie=UTF-8).

I think you forgot to mention specific details in your report but based on google search results for the error text it's safe to say that it happens when running the tool on node v8. There is an open issue in the dependency I'm using, [daemon](https://github.com/indexzero/daemon.node/issues/41). The fix isn't merged yet, but across other repos there is a shared common [remedy](https://github.com/NodeBB/NodeBB/issues/5780#issuecomment-311263357):
> indexzero/daemon.node#41
For anyone else affected, a quick workaround is to apply zipang's PR:
npm install 'github:zipang/daemon.node#48d0977c26fb3a6a44ae99aae3471b9d5a761085'

I tried it and it fixed the issue when using node v8.

**Comment by simsalabim** [comment]
I applied the [patch](https://github.com/simsalabim/dsweeper/commit/bbfd0ec114eb822fb6d3635bfe7dfbd42a4e855c) and published the new version to npm, [dsweeper@0.0.8](https://www.npmjs.com/package/dsweeper).

**Comment by ghost** [comment]
Oh it was OK and it works fine
